.vaunt/config.yaml

```diff
@@ -32,3 +32,21 @@ achievements:
               actor: reviewers
               action: pull_request
               condition: merged = true
+    - achievement: 
+      name: Tqla Honey Bee
+      icon: https://raw.githubusercontent.com/VauntDev/tqla/main/.vaunt/tqla_honey_bee.png
+      description: We bee-lieve in you! This achievement celebrates your first bug report, helping make Tqla sweeter than ever.
+      triggers:
+        - trigger:
+            actor: author
+            action: issue
+            condition: labels in ['bug']
```

Comment:
Can you update the conditions for these new achievements to require an action from one of the codeowners?

I think you could do something like this:

```
labels in ['bug'] & reactors in ['elewis787', 'jeff10110322', 'simonmazzaroth']
```